azurerm/internal/services/apimanagement/validate/api_management_test.go

```diff
@@ -0,0 +1,82 @@
+package validate
+
+import "testing"
+
+func TestApimSkuName(t *testing.T) {
+	tests := []struct {
+		name  string
+		input string
+		valid bool
+	}{
+		{
+			name:  "Consumption_0",
+			input: "Consumption_0",
+			valid: true,
+		},
+		{
+			name:  "Consumption_1",
+			input: "Consumption_1",
+			valid: false,
+		},
+		{
+			name:  "Basic_3",
+			input: "Basic_3",
+			valid: false,
+		},
+		{
+			name:  "Basic_1",
+			input: "Basic_1",
+			valid: true,
+		},
+		{
+			name:  "Developer_1",
+			input: "Developer_1",
+			valid: true,
+		},
+		{
+			name:  "Premium_0",
+			input: "Premium_0",
+			valid: false,
+		},
+		{
+			name:  "Premium_11",
+			input: "Premium_11",
+			valid: false,
+		},
+		{
+			name:  "Premium_7",
+			input: "Premium_7",
+			valid: true,
+		},
+		{
+			name:  "Standard_7",
+			input: "Standard_7",
+			valid: false,
+		},
+		{
+			name:  "Standard_2",
+			input: "Standard_2",
+			valid: true,
+		},
+		{
+			name:  "standard_2",
+			input: "standard_2",
+			valid: true,
+		},
+		{
+			name:  "PREMIUM_7",
+			input: "PREMIUM_7",
```

Comment:
why is this one allin uppercase?

Reply:
Because case ignored is accepted for api.